dse-k8s-operator/README.md

```diff
@@ -396,6 +446,10 @@ this time.
 
 # Changelog
 
+## v0.9.0
+
+* The operator can work with OSS Apache Cassandra.
```

Comment:
This seems a bit misleading. I would include the migration to `CassandraDatacenter` and the addition of super user secrets. Since this is going to be OSS something mentioning the new integration tests should be included as well.